tests/Api/CampaignsTest.php

```diff
@@ -471,7 +471,7 @@ public function testCampaignContactEditEvent()
                 $date = new \DateTime($log['triggerDate'], new \DateTimeZone('UTC'));
                 $this->assertEquals($log['triggerDate'], $date->format('c'));
             } else {
-                $this->assertFalse(false, 'Event ID not recognized in the log.', var_export($event, true));
+                $this->assertFalse(var_export($event, true), 'Event ID not recognized in the log.');
```

Comment:
This doesn't make much sense. The previous version did not make sense either. I expect the intent was to fail the test if this assert will run. How about this:
```suggestion
                $this->fail('Event ID not recognized in the log.'.var_export($event, true));
```

Reply:
@dennisameling WDYT?